cves/CVE-2014-0231.yml

```diff
@@ -177,38 +207,43 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
-    note:
+    applies: true
+    note: |
+        timeouts are a layer of robustness underneath normal CGI input validation
   least_privilege:
-    applies:
+    applies: false
     note:
   frameworks_are_optional:
-    applies:
+    applies: false
     note:
   native_wrappers:
-    applies:
+    applies: false
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true 
+    note: |
+      centered around handling abnormal input (input which causes the script 
+      to halt)
```

Comment:
I think that having a full sentence and a little more detail here would be important for the CWE.